cmd/checks/components/components_check.go

```diff
@@ -69,6 +70,7 @@ func Add(root *cobra.Command) {
 	componentsCmd.Flags().StringVarP(&healthURLPrefix, "health-url", "u", "/system/health/v1", "Set dcos-diagnostics health url")
 	componentsCmd.Flags().StringVarP(&scheme, "scheme", "s", "http", "Set dcos-diagnostics health url scheme")
 	componentsCmd.Flags().IntVarP(&port, "port", "p", 1050, "Set TCP port")
+	componentsCmd.Flags().StringArrayVar(&excludeComponents, "exclude", nil, "Exclude components from health check")
```

Comment:
i'd still want the options to be consistent, it should be `StringArrayVarP` as the rest of the flags, or you need to refactor the rest without `P`

Reply:
done